### ISSUE TITLE
PDF: Display of attributions

### ISSUE THREAD

**Issue by alex-418** [BCcampus/pressbooks#6]
Displayed at the end of the chapter, each attribution doesn't have to be on the same page as the image. With the title "Attributions" in a "textbox". The goal is to adhere to accessibility standards.
